### PR TITLE
Fdl message

### PR DESCRIPTION
**Pull request by gstenson**

### CODE REVIEW

**Comment by gstenson** [comment]
FDL message as required by BNP. French banks seem to use a different subset of messages.

**Comment by bumi** [comment]
@gstenson thanks for your contribution. @kangguru do you have time to review this PR?

File: lib/epics/fdl.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+          }
+          xml.SecurityMedium "0000"
+        }
+        xml.mutable {
+          xml.TransactionPhase 'Initialisation'
+        }
+      }
+    end
+    xml_string = builder.to_xml 
+    File.open('/tmp/file.xml', 'w') do |file|
```
Comment (kangguru) [inline lib/epics/fdl.rb:40]: @gstenson why do you need to write the xml as a file?

Comment (gstenson) [reply lib/epics/fdl.rb:40]: Ooops! Hangover from outputting in development. Will resolve

Comment (kangguru) [reply lib/epics/fdl.rb]: Cool!

**Comment by frantisekrokusek** [comment]
@tobischo I think we can close this one 😜

**Comment by tobischo** [comment]
Indeed, thanks for pointing that out

**Pull request closed without merging**